src/transformers/utils/generic.py

```diff
@@ -123,6 +123,24 @@ def is_torch_device(x):
     return False if not is_torch_available() else _is_torch_device(x)
 
 
+def _is_torch_dtype(x):
+    import torch
+
+    if isinstance(x, str):
+        if hasattr(torch, x):
+            return True
+        else:
+            return False
+    return isinstance(x, torch.dtype)
+
+
+def is_torch_dtype(x):
+    """
+    Tests if `x` is a torch device or not. Safe to call even if torch is not installed.
```

Comment:
device -> dtype ?

Reply:
Yes ideed :D fixed in [ba94418](https://github.com/huggingface/transformers/pull/20536/commits/ba9441879c16f882dc55320773d86c30295f438f)